src/libcore/mem.rs

```diff
@@ -359,7 +358,7 @@ pub fn align_of_val<T: ?Sized>(val: &T) -> usize {
 /// }
 /// ```
 #[inline]
-#[unstable(feature = "needs_drop", issue = "41890")]
+#[stable(feature = "needs_drop", since = "1.22.0")]
```

Comment:
If this was released with Rust 1.21, is the since field wrong?

Reply:
@oherrala The versions has been fixed in 874124b2c73dc764aa83f7ba9711a4170c0aeffc 🙂

Reply:
@kennytm, ah, cool. Sorry about the noise.